srcpkgs/glibc/patches/committed-1-2-Propagate-GLIBC_TUNABLES-in-setxid-binaries.patch

```diff
@@ -0,0 +1,417 @@
+From patchwork Tue Oct  3 17:08:10 2023
```

Comment:
do we need to include all the email headers?

Reply:
Removed most of the headers.